Flaky test:  TestInverse

[Jenkins](https://jenkins.preferred.jp/job/chainer/job/chainer_pr/1963/)

tests/chainerx_tests/unit_tests/routines_tests/test_linalg.py::test_Inverse_param_1_{dtype='float32', shape=(3, 3)}[cuda:0]

```
_________ test_Inverse_param_1_{dtype='float32', shape=(3, 3)}[cuda:0] _________

device = cuda:0, args = (), kwargs = {}
backend_config = <BackendConfig use_chainerx=True chainerx_device='cuda:0' use_cuda=False cuda_device=None use_cudnn='never' cudnn_deterministic=False autotune=False cudnn_fast_batch_normalization=False use_ideep='never'>
obj = <chainer.testing._bundle.TestInverse_param_1_{dtype='float32', shape=(3, 3)} object at 0x7f1678cce278>

    @pytest.mark.parametrize_device(devices)
    def entry_func(device, *args, **kwargs):
        backend_config = _make_backend_config(device.name)
    
        # Forward test
        obj = cls()
        try:
            obj.setup(*args, **kwargs)
>           obj.run_test_forward(backend_config)

args       = ()
backend_config = <BackendConfig use_chainerx=True chainerx_device='cuda:0' use_cuda=False cuda_device=None use_cudnn='never' cudnn_deterministic=False autotune=False cudnn_fast_batch_normalization=False use_ideep='never'>
cls        = <class 'chainer.testing._bundle.TestInverse_param_1_{dtype='float32', shape=(3, 3)}'>
device     = cuda:0
kwargs     = {}
obj        = <chainer.testing._bundle.TestInverse_param_1_{dtype='float32', shape=(3, 3)} object at 0x7f1678cce278>

/repo/tests/chainerx_tests/op_utils.py:352: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/repo/tests/chainerx_tests/op_utils.py:88: in run_test_forward
    super(OpTest, self).run_test_forward(backend_config)
/workspace/conda/envs/testenv/lib/python3.6/site-packages/chainer/testing/function_link.py:211: in run_test_forward
    cpu_expected)
/repo/tests/chainerx_tests/op_utils.py:301: in check_forward_outputs
    outputs, expected_outputs)
/workspace/conda/envs/testenv/lib/python3.6/site-packages/chainer/testing/function_link.py:123: in check_forward_outputs
    **self.check_forward_options)
/workspace/conda/envs/testenv/lib/python3.6/site-packages/chainer/testing/function_link.py:1229: in _check_arrays_equal
    test_error_cls.fail(msg)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

cls = <class 'chainer.testing.function_link.FunctionTestError'>
message = 'Outputs do not match the expected values.\nIndices of outputs that do not match: 0\nExpected shapes and dtypes: (3, 3...23  -365.2334     83.20899]\n    [ 3669.4783  -5676.943    1294.7919 ]\n    [-3747.6821   5795.3604  -1320.6923 ]]\n\n'
exc = None

    @classmethod
    def fail(cls, message, exc=None):
        if exc is not None:
            utils._raise_from(cls, message, exc)
>       raise cls(message)
E       chainer.testing.function_link.FunctionTestError: Outputs do not match the expected values.
E       Indices of outputs that do not match: 0
E       Expected shapes and dtypes: (3, 3):float32
E       Actual shapes and dtypes:   (3, 3):float32
E       
E       
E       Error details of output [0]:
E       
E       Not equal to tolerance rtol=0.0001, atol=1e-05
E       
E       Mismatch: 100%
E       Max absolute difference: 0.6074219
E       Max relative difference: 0.00010628
E        x: array([[  238.37651,  -365.19458,    83.20027],
E              [ 3669.0935 , -5676.348  ,  1294.656  ],
E              [-3747.2893 ,  5794.753  , -1320.5536 ]], dtype=float32)
E        y: array([[  238.40123,  -365.2334 ,    83.20899],
E              [ 3669.4783 , -5676.943  ,  1294.7919 ],
E              [-3747.6821 ,  5795.3604 , -1320.6923 ]], dtype=float32)
E       
E       assert_allclose failed: 
E         shape: (3, 3) (3, 3)
E         dtype: float32 float32
E         i: (2, 1)
E         x[i]: 5794.7529296875
E         y[i]: 5795.3603515625
E         relative error[i]: 0.00010481175559107214
E         absolute error[i]: 0.607421875
E         relative tolerance * |y[i]|: 0.5795360351562501
E         absolute tolerance: 1e-05
E         total tolerance: 0.57954603515625
E       x: [[  238.37651  -365.19458    83.20027]
E           [ 3669.0935  -5676.348    1294.656  ]
E           [-3747.2893   5794.753   -1320.5536 ]]
E       y: [[  238.40123  -365.2334     83.20899]
E           [ 3669.4783  -5676.943    1294.7919 ]
E           [-3747.6821   5795.3604  -1320.6923 ]]

cls        = <class 'chainer.testing.function_link.FunctionTestError'>
exc        = None
message    = 'Outputs do not match the expected values.\nIndices of outputs that do not match: 0\nExpected shapes and dtypes: (3, 3...23  -365.2334     83.20899]\n    [ 3669.4783  -5676.943    1294.7919 ]\n    [-3747.6821   5795.3604  -1320.6923 ]]\n\n'

/workspace/conda/envs/testenv/lib/python3.6/site-packages/chainer/testing/function_link.py:28: FunctionTestError
_ test_Min_param_105_{in_dtypes=('float16',), out_dtype='float16', is_module=False, shape=(4, 2), axis=-2}[native:0] _

device = native:0, args = (), kwargs = {}
backend_config = <BackendConfig use_chainerx=True chainerx_device='native:0' use_cuda=False cuda_device=None use_cudnn='never' cudnn_deterministic=False autotune=False cudnn_fast_batch_normalization=False use_ideep='never'>
obj = <chainer.testing._bundle.TestMin_param_105_{in_dtypes=('float16',), out_dtype='float16', is_module=False, shape=(4, 2), axis=-2} object at 0x7f168acac240>

    @pytest.mark.parametrize_device(devices)
    def entry_func(device, *args, **kwargs):
        backend_config = _make_backend_config(device.name)
    
        # Forward test
        obj = cls()
        try:
            obj.setup(*args, **kwargs)
            obj.run_test_forward(backend_config)
        finally:
            obj.teardown()
    
        # If this is a NumpyOpTest instance, skip backward/double-backward
        # tests if the forward test succeeds with acceptable errors.
        if isinstance(obj, NumpyOpTest):
            if obj.is_forward_successful_with_accept_errors:
                return  # success with expected errors
    
        # Backward test
        obj = cls()
        try:
            obj.setup(*args, **kwargs)
>           obj.run_test_backward(backend_config)

args       = ()
backend_config = <BackendConfig use_chainerx=True chainerx_device='native:0' use_cuda=False cuda_device=None use_cudnn='never' cudnn_deterministic=False autotune=False cudnn_fast_batch_normalization=False use_ideep='never'>
cls        = <class 'chainer.testing._bundle.TestMin_param_105_{in_dtypes=('float16',), out_dtype='float16', is_module=False, shape=(4, 2), axis=-2}'>
device     = native:0
kwargs     = {}
obj        = <chainer.testing._bundle.TestMin_param_105_{in_dtypes=('float16',), out_dtype='float16', is_module=False, shape=(4, 2), axis=-2} object at 0x7f168acac240>

/repo/tests/chainerx_tests/op_utils.py:366: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/repo/tests/chainerx_tests/op_utils.py:95: in run_test_backward
    super(OpTest, self).run_test_backward(backend_config)
/workspace/conda/envs/testenv/lib/python3.6/site-packages/chainer/testing/function_link.py:249: in run_test_backward
    do_check()
/workspace/conda/envs/testenv/lib/python3.6/site-packages/chainer/testing/function_link.py:244: in do_check
    **self.check_backward_options)
/workspace/conda/envs/testenv/lib/python3.6/contextlib.py:99: in __exit__
    self.gen.throw(type, value, traceback)
/workspace/conda/envs/testenv/lib/python3.6/site-packages/chainer/testing/function_link.py:36: in raise_if_fail
    cls.fail(message, e)
/workspace/conda/envs/testenv/lib/python3.6/site-packages/chainer/testing/function_link.py:27: in fail
    utils._raise_from(cls, message, exc)
/workspace/conda/envs/testenv/lib/python3.6/site-packages/chainer/utils/__init__.py:106: in _raise_from
    six.raise_from(new_exc.with_traceback(orig_exc.__traceback__), None)
<string>:3: in raise_from
    ???
/workspace/conda/envs/testenv/lib/python3.6/site-packages/chainer/testing/function_link.py:34: in raise_if_fail
    yield
/workspace/conda/envs/testenv/lib/python3.6/site-packages/chainer/testing/function_link.py:244: in do_check
    **self.check_backward_options)
/workspace/conda/envs/testenv/lib/python3.6/site-packages/chainer/gradient_check.py:900: in check_backward
    detect_nondifferentiable, is_immutable_params=False
/workspace/conda/envs/testenv/lib/python3.6/site-packages/chainer/gradient_check.py:464: in run
    self._run()
/workspace/conda/envs/testenv/lib/python3.6/site-packages/chainer/gradient_check.py:507: in _run
    self._compare_gradients(gx_numeric, gx_backward, directions)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <chainer.gradient_check._CheckBackward object at 0x7f168acac860>
gx_numeric = array(-0.33893517, shape=(), dtype=float64, device='native:0')
gx_backward = array(-0.57545740, shape=(), dtype=float64, device='native:0')
directions = [array([[0.58140456, -0.47981064],
       [-0.18558059, -0.16559088],
       [-0.40996093, -0.24717295],
       [-0.27239242, -0.25793396]], shape=(4, 2), dtype=float64, device='native:0')]

    def _compare_gradients(self, gx_numeric, gx_backward, directions):
        atol = self.atol
        rtol = self.rtol
        # Compare the gradients
        try:
            testing.assert_allclose(
                gx_numeric, gx_backward, atol=atol, rtol=rtol)
        except AssertionError as e:
            eps = self.eps
            xs = self.xs
            gys = self.gys
            f = six.StringIO()
            f.write('check_backward failed (eps={} atol={} rtol={})\n'.format(
                eps, atol, rtol))
            for i, x in enumerate(xs):
                f.write('inputs[{}]:\n'.format(i))
                f.write('{}\n'.format(x))
            for i, gy in enumerate(gys):
                f.write('grad_outputs[{}]:\n'.format(i))
                f.write('{}\n'.format(gy))
            for i, d in enumerate(directions):
                f.write('directions[{}]:\n'.format(i))
                f.write('{}\n'.format(d))
            f.write('gradients (numeric):  {}\n'.format(gx_numeric))
            f.write('gradients (backward): {}\n'.format(gx_backward))
            f.write('\n')
            f.write('x: numeric gradient, y: backward gradient')
            f.write(str(e))
>           raise AssertionError(f.getvalue())
E           chainer.testing.function_link.FunctionTestError: backward is not implemented correctly
E           
E           (caused by)
E           AssertionError: check_backward failed (eps=0.001 atol=0.003 rtol=0.003)
E           inputs[0]:
E           array([[-0.5180664 , -0.18029785],
E                  [0.17419434, 0.70507812],
E                  [-0.08709717, -0.73535156],
E                  [-0.08740234, -0.73535156]], shape=(4, 2), dtype=float16, device='native:0')
E           grad_outputs[0]:
E           array([-0.17614746, 0.93652344], shape=(2,), dtype=float16, device='native:0')
E           directions[0]:
E           array([[0.58140456, -0.47981064],
E                  [-0.18558059, -0.16559088],
E                  [-0.40996093, -0.24717295],
E                  [-0.27239242, -0.25793396]], shape=(4, 2), dtype=float64, device='native:0')
E           gradients (numeric):  array(-0.33893517, shape=(), dtype=float64, device='native:0')
E           gradients (backward): array(-0.57545740, shape=(), dtype=float64, device='native:0')
E           
E           x: numeric gradient, y: backward gradient
E           Not equal to tolerance rtol=0.003, atol=0.003
E           
E           Mismatch: 100%
E           Max absolute difference: 0.23652223
E           Max relative difference: 0.41101606
E            x: array(-0.338935)
E            y: array(-0.575457)
E           
E           assert_allclose failed: 
E             shape: () ()
E             dtype: float64 float64
E             i: (0,)
E             x[i]: -0.3389351676931018
E             y[i]: -0.5754573976293433
E             relative error[i]: 0.4110160559419681
E             absolute error[i]: 0.23652222993624156
E             relative tolerance * |y[i]|: 0.00172637219288803
E             absolute tolerance: 0.003
E             total tolerance: 0.00472637219288803
E           x: -0.33893517
E           y: -0.5754574

atol       = 0.003
d          = array([[0.58140456, -0.47981064],
       [-0.18558059, -0.16559088],
       [-0.40996093, -0.24717295],
       [-0.27239242, -0.25793396]], shape=(4, 2), dtype=float64, device='native:0')
directions = [array([[0.58140456, -0.47981064],
       [-0.18558059, -0.16559088],
       [-0.40996093, -0.24717295],
       [-0.27239242, -0.25793396]], shape=(4, 2), dtype=float64, device='native:0')]
eps        = 0.001
f          = <_io.StringIO object at 0x7f1667c3fdc8>
gx_backward = array(-0.57545740, shape=(), dtype=float64, device='native:0')
gx_numeric = array(-0.33893517, shape=(), dtype=float64, device='native:0')
gy         = array([-0.17614746, 0.93652344], shape=(2,), dtype=float16, device='native:0')
gys        = (array([-0.17614746, 0.93652344], shape=(2,), dtype=float16, device='native:0'),)
i          = 0
rtol       = 0.003
self       = <chainer.gradient_check._CheckBackward object at 0x7f168acac860>
x          = array([[-0.5180664 , -0.18029785],
       [0.17419434, 0.70507812],
       [-0.08709717, -0.73535156],
       [-0.08740234, -0.73535156]], shape=(4, 2), dtype=float16, device='native:0')
xs         = (array([[-0.5180664 , -0.18029785],
       [0.17419434, 0.70507812],
       [-0.08709717, -0.73535156],
       [-0.08740234, -0.73535156]], shape=(4, 2), dtype=float16, device='native:0'),)
```

--- a/tests/chainerx_tests/unit_tests/routines_tests/test_linalg.py
+++ b/tests/chainerx_tests/unit_tests/routines_tests/test_linalg.py
@@ -212,6 +212,7 @@ class TestInverse(NumpyLinalgOpTest):
 
     def generate_inputs(self):
         a = numpy.random.random(self.shape).astype(self.dtype)
+        a = a * 10 + numpy.ones(self.shape)
         return a,
 
     def forward_xp(self, inputs, xp):

